.devcontainer/devcontainer.json

```diff
@@ -4,7 +4,25 @@
     "dockerfile": "Dockerfile",
     "context": ".."
   },
-  "appPort": "8124:8123",
+  "forwardPorts": [8123, 8100, 8090, 8080],
+  "portsAttributes": {
+    "8123": {
+      "label": "Home Assistant",
+      "onAutoForward": "silent"
```

Comment:
Why is "silent" added here (and for the others)?
The default is "notify" which is quite nice.

Reply:
It's just my preference, I'll change it. I should work on this PR tonight.